docs/application/native/guides/text-input/autofill-service.md

```diff
@@ -0,0 +1,328 @@
+# Autofill Service
+
+Autofill Service application is a service application to store the user input and provide the data previously entered by the user.
+Using the autofill service functions, Autofill service application developers can implement their own autofill service to serve the autofill information.
+
+The main features of the Autofill service API include:
+
+- [Receiving the request of authentication information and sending the authentication information](#receiving-the-request-of-authentication-information-and-sending-the-authentication-information)
+
+  To receive the request of authentication information, use the `autofill_service_set_auth_info_requested_cb()` function. The autofill authentication information can be sent with `autofill_service_send_auth_info()`.
+
+- [Receiving the request of autofill fill request and sending the autofill fill response](#receiving-the-request-of-autofill-fill-request-and-sending-the-autofill-fill-response)
```

Comment:
- [Receiving request of autofill fill request and sending the autofill fill response](#receiving-the-request-of-autofill-fill-request-and-sending-the-autofill-fill-response)